lib/Install.js

```diff
@@ -117,6 +118,44 @@ function install(
   return 'SuccessfullyInstalled';
 }
 
+async function installUnstableTestMaster() {
+  const tempDir = 'elm-stuff/elm-build-cache';
+  const pkg = 'elm-explorations/test';
+  const version = '1.2.2';
+  const pkgWithDash = pkg.replace('/','-');
+
+  const zipballUrl = `https://github.com/${pkg}/archive/refs/heads/master.zip`;
+  const zipballPath = `${tempDir}/${pkgWithDash}.zip`;
+
+  // based on info in https://package.elm-lang.org/packages/elm/project-metadata-utils/latest/
+  const elmHome = process.env.ELM_HOME ?? process.platform === 'win32' ? `${process.env.APPDATA}/elm` : `~/.elm`;
+  const packagePath = `${elmHome}/0.19.1/packages/${pkg}/${version}`;
+
+  const ourExec = async (cmd) => 
+    new Promise((resolve, reject) => {
+      exec(cmd, (error,stdout,stderr) => {
+        if (error) {
+          reject(error);
+        }
+        resolve([stdout,stderr]);
+      });
+    });
+
+  await ourExec(`rm -rf ${tempDir}`);
+  await ourExec(`mkdir -p ${tempDir}`);
+  await ourExec(`rm -rf ${packagePath}`);
+  await ourExec(`rm -rf elm-stuff/generated-code`);
```

Comment:
Should this only delete `elm-stuff/generated-code/elm-community/elm-test`? To avoid deletin elm-review’s generated code for example.

Also, all paths should be written based on the “root dir” like in https://github.com/rtfeldman/node-test-runner/blob/fd52f043b9ae6ce20596af123af3be1a7967a383/lib/Project.js#L46-L53